compiler/daml-extension/syntaxes/daml.json

```diff
@@ -117,7 +117,7 @@
         },
         {
           "name": "keyword.control.daml",
```

Comment:
Hm, do we want to stop highlighting scenario or should we just highlight both?

Reply:
Hm probably better to keep it for now, I added it back